pysparkling/__version__.py

```diff
@@ -0,0 +1,3 @@
+from ._version import get_versions
```

Comment:
Yes indeed. I couldn't fix it in another way because it would recursively call the library and as the first step the version again.

Reply:
I have to come back to this. In `context.py` this also happened before: Context is imported from `__init__.py` but inside of `context.py` the `__version__` is imported from `__init__.py`. 

I assume because of the order that `__version__` was the first thing defined, this doesn't lead to a cycle. This should still be possible.